Source/Core/LMCConst.hh

```diff
@@ -70,12 +70,12 @@ public:
     //masses
     constexpr static double M_el_kg()
     {
-        return 9.1093826E-31;
+        return 9.1093837015E-31; // updated from https://pdg.lbl.gov/2021
     } //!< electron mass in kg
 
     constexpr static double M_el_eV()
     {
-        return 510.998918E3;
+        return 510.998950E3; // updated from https://pdg.lbl.gov/2021
```

Comment:
I would write it as `510.99895000E3` (with three zeros at the end) as this is the precision with which the quantity is known and reported.